cypress/integration/landing.spec.ts

```diff
@@ -0,0 +1,119 @@
+      });
+    });
+
+    it('Switch theme should change to corresponding examples', () => {
+      themeOptions.forEach((themeOption, i) => {
+        const themeId = themeOption.id;
+        cy.get('#theme-selector').click();
+        cy.get(`#react-select-theme-selector-option-${i}`).click();
+        cy.get('.search-example').eq(0).should('have.text', examples[themeId].left[0].text);
+        cy.get('.search-example').eq(1).should('have.text', examples[themeId].left[1].text);
+        cy.get('.search-example').eq(2).should('have.text', examples[themeId].left[2].text);
+        cy.get('.search-example').eq(3).should('have.text', examples[themeId].left[3].text);
+        cy.get('.search-example').eq(4).should('have.text', examples[themeId].left[4].text);
+        cy.get('.search-example').eq(5).should('have.text', examples[themeId].left[5].text);
+        cy.get('.search-example').eq(6).should('have.text', examples[themeId].left[6].text);
+        cy.get('.search-example').eq(7).should('have.text', examples[themeId].right[0].text);
+        cy.get('.search-example').eq(8).should('have.text', examples[themeId].right[1].text);
+        cy.get('.search-example').eq(9).should('have.text', examples[themeId].right[2].text);
+        cy.get('.search-example').eq(10).should('have.text', examples[themeId].right[3].text);
+        cy.get('.search-example').eq(11).should('have.text', examples[themeId].right[4].text);
+        cy.get('.search-example').eq(12).should('have.text', examples[themeId].right[5].text);
+      });
+    });
+
+    it('Classic and paper forms should only be available on Astrophysics', () => {
+      cy.get('#theme-selector').click();
+      cy.get('#react-select-theme-selector-option-0').click();
+      cy.get('a[href="/classic-form"]').should('not.exist');
+      cy.get('a[href="/paper-form"]').should('not.exist');
+      cy.get('#theme-selector').click();
+      cy.get('#react-select-theme-selector-option-1').click();
+      cy.get('a[href="/classic-form"]').should('exist');
+      cy.get('a[href="/paper-form"]').should('exist');
+      cy.get('#theme-selector').click();
+      cy.get('#react-select-theme-selector-option-2').click();
+      cy.get('a[href="/classic-form"]').should('not.exist');
+      cy.get('a[href="/paper-form"]').should('not.exist');
+      cy.get('#theme-selector').click();
+      cy.get('#react-select-theme-selector-option-3').click();
+      cy.get('a[href="/classic-form"]').should('not.exist');
+      cy.get('a[href="/paper-form"]').should('not.exist');
+      cy.get('#theme-selector').click();
+      cy.get('#react-select-theme-selector-option-4').click();
+      cy.get('a[href="/classic-form"]').should('not.exist');
+      cy.get('a[href="/paper-form"]').should('not.exist');
+      cy.get('#theme-selector').click();
+      cy.get('#react-select-theme-selector-option-5').click();
+      cy.get('a[href="/classic-form"]').should('not.exist');
+      cy.get('a[href="/paper-form"]').should('not.exist');
```

Comment:
Same as above, if the ordering were to change or something it would break.  Maybe that's okay, but in this test we're only really concerned with whether classic/paper forms disappear when `theme !== 'astrophysics'`.
I think we could simplify and make this test more resistant to change

Reply:
These have been changed so that I am not making any assumptions about the theme ordering.